packages/dashboard/src/customization/propertiesSections/lineAndScatterStyleSettings/lineStyleSection.tsx

```diff
@@ -1,36 +1,49 @@
 import React from 'react';
 import type { FC } from 'react';
-
 import ExpandableSection from '@cloudscape-design/components/expandable-section';
-
 import { LineStyleDropdown } from '../components/lineStyleDropdown';
 import { LineThicknessDropdown } from '../components/lineThicknessDropdown';
-import { SpaceBetween } from '@cloudscape-design/components';
+import { Box, SpaceBetween } from '@cloudscape-design/components';
+import { LineTypeSection } from '../components/lineType';
+import { DataPointStyleSection } from './dataPointStyleSection';
+import './lineAndScatterStyleSettings.css';
 
 type LineStyleSectionOptions = {
+  lineType?: string;
   lineStyle?: string;
   lineThickness?: number;
+  dataPointStyle: string | undefined;
+  updateType: (lineType: string) => void;
   updatelineStyle: (lineStyle: string) => void;
   updateLineThickness: (lineThickness: number) => void;
+  updateDataPointStyle: (dataPointStyle: string) => void;
 };
 
 export const LineStyleSection: FC<LineStyleSectionOptions> = ({
+  lineType,
   lineStyle,
   lineThickness,
+  dataPointStyle,
+  updateType,
   updatelineStyle,
   updateLineThickness,
+  updateDataPointStyle,
 }) => {
   return (
-    <ExpandableSection headerText='Line style' defaultExpanded>
-      <SpaceBetween size='m'>
-        <LineStyleDropdown lineStyle={lineStyle} updatelineStyle={updatelineStyle} />
-        <LineThicknessDropdown
-          lineThickness={lineThickness?.toString() ?? '2'}
-          updateLineThickness={(thickness: string) => {
-            updateLineThickness(parseInt(thickness));
-          }}
-        />
-      </SpaceBetween>
+    <ExpandableSection className='accordian-header' headerText='Widget style' defaultExpanded variant='footer'>
+      <Box padding='s'>
+        <SpaceBetween size='m'>
+          <LineTypeSection type={lineType} updateType={updateType} />
+          <LineStyleDropdown lineStyle={lineStyle} updatelineStyle={updatelineStyle} />
+          <LineThicknessDropdown
+            lineThickness={lineThickness?.toString() ?? '2'}
```

Comment:
worth creating a const for '2' or re-using if there is a constant already

Reply:
resolved